### PR TITLE
[MM-36879] Add additional developer setup instructions

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
update README with instructions for connecting MM server to zendesk cloud

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-36879

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,10 +106,14 @@ Creating a ticket from a Mattermost post is done through the `...` post menu but
 
 ## Development
 
+Mattermost Server Setup
+
+To allow the Mattermost server to communicate with a Zendesk cloud instance, you must expose your local mattermost instance via a tool such as ngrok.io. Set the `MM_SERVICESETTINGS_SITEURL` environment variable to this exposed URL and restart your mattermost server. You can ensure the url is correct via `System Console` > `ENVIRONMENT` > `Web Server` > `Site URL`
```
Comment (mickmister) [inline README.md:111]:
```suggestion
To allow the Mattermost server to communicate with a Zendesk cloud instance, your Mattermost instance needs to be exposed to the internet via a tool such as [ngrok.io](https://ngrok.io). The Mattermost server then needs to be configured to use the exposed address. You can set it in the system console at `System Console` > `ENVIRONMENT` > `Web Server` > `Site URL`, or set the `MM_SERVICESETTINGS_SITEURL` environment variable. You'll need to restart your Mattermost server after you change the Site URL. You can ensure the URL is correct by clicking "Test Live URL" in the system console page mentioned above.
```

**Review by mickmister — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jfrerich) [reply README.md]: Much better.  Thank you sir :)

File: README.md [hunk 2]
```diff
@@ -106,10 +106,14 @@ Creating a ticket from a Mattermost post is done through the `...` post menu but
...
+
 To install Zendesk in a development environment:
 
 `/apps debug-add-manifest --url http://localhost:4000/manifest.json`  
-`/apps install --app-id com.mattermost.zendesk --app-secret thisisthesecret`
+`/apps install http com.mattermost.zendesk`
```
[the remark this answers is not in the document]
Comment (jfrerich) [reply README.md:116]: You are correct.  An `http` install uses the URL and an `aws` install uses the pluginID.  For the install instructions, I should probably use the AWS instructions, correct?

[the remark this answers is not in the document]
Comment (mickmister) [reply README.md:116]: For installing in a development environment (what this section describes), I suggest using the http install version. If I were to add any features to this project, I would want to use the http mode so I can be up and running quick. What value does the aws deployment give the developer that is working on the App?

[the remark this answers is not in the document]
Comment (jfrerich) [reply README.md:116]: 👍 you're correct.  This is for development and should use `http` not `aws`.

**Review by mickmister — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -106,10 +106,14 @@ Creating a ticket from a Mattermost post is done through the `...` post menu but
 
 ## Development
 
+Mattermost Server Setup
+
+To allow the Mattermost server to communicate with a Zendesk cloud instance, your Mattermost instance needs to be exposed to the internet via a tool such as [ngrok.io](https://ngrok.io). The Mattermost server then needs to be configured to use the exposed address. You can set it in the system console at `System Console` > `ENVIRONMENT` > `Web Server` > `Site URL`, or set the `MM_SERVICESETTINGS_SITEURL` environment variable. You'll need to restart your Mattermost server after you change the Site URL. You can ensure the URL is correct by clicking "Test Live URL" in the system console page mentioned above.
```
Comment (justinegeffen) [inline README.md:111]:
```suggestion
To allow the Mattermost server to communicate with a Zendesk cloud instance, your Mattermost instance needs to be exposed to the internet via a tool such as [ngrok.io](https://ngrok.io). The Mattermost server then needs to be configured to use the exposed address. You can set it via **System Console > ENVIRONMENT > Web Server > Site URL**. Then select **Test Live URL** to confirm that the URL is correct. You can also set the `MM_SERVICESETTINGS_SITEURL` environment variable. 

You'll need to restart your Mattermost server after you change the Site URL. 
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**